CHANGELOG.md

```diff
@@ -1,3 +1,49 @@
+OSV-Scanner v2 is coming soon! The next release will start with version `v2.0.0-alpha1`.
+
+Here's a peek at some of the exciting upcoming features:
+
+- Standalone container image scanning support.
+  - Including support for Alpine and Debian images.
```

Comment:
remove space?

Reply:
Do you mean the indent? That is intentional

Reply:
yes